lib/davinci_dtr_test_kit/profiles/questionnaire_response_group.rb

```diff
@@ -0,0 +1,76 @@
+  class QuestionnaireResponseGroup < Inferno::TestGroup
+    title 'DTR QuestionnaireResponse Tests'
+    short_description 'Verify support for the server capabilities required by the DTR QuestionnaireResponse Profile'
+    description %(
+      # Background
+
+    The DTR QuestionnaireResponse sequence verifies that the system under test is
+    able to provide correct responses for QuestionnaireResponse queries. These queries
+    must contain resources conforming to the DTR QuestionnaireResponse Profile as
+    specified in the DaVinci Documentation Templates and Rules (DTR) v2.0.1
+    Implementation Guide.
+
+    # Testing Methodology
+    ## Searching
+    This test sequence will first perform each required search associated
+    with this resource. This sequence will perform searches with the
+    following parameters:
+
+    * patient + context
+
+    ### Search Parameters
+    The search step uses the selected patient(s) from the prior launch
+    sequence. Any subsequent searches will look for its parameter values
+    from the results of the search step. For example, the `identifier`
+    search in the patient sequence is performed by looking for an existing
+    `Patient.identifier` from any of the resources returned in the `_id`
+    search. If a value cannot be found this way, the search is skipped.
+
+    ### Search Validation
+    Inferno will retrieve up to the first 20 bundle pages of the reply for
+    QuestionnaireResponse resources and save them for subsequent tests. Each of
+    these resources is then checked to see if it matches the searched
+    parameters in accordance with [FHIR search
+    guidelines](https://www.hl7.org/fhir/search.html). The test will fail,
+    for example, if a Patient search for `gender=male` returns a `female`
+    patient.
+
+    ## Read
+    This test sequence will first perform the required read associated
+    with this resource. The server SHALL be capable of returning a
+    QuestionnaireResponse resource using the read interaction.
+
+    ## Profile Validation
+    Each resource returned from the read step SHALL conform to
+    the [DTR QuestionnaireResponse Profile](http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaireresponse).
+    Each element is checked against terminology binding and cardinality requirements.
+
+    ## Create
+    This test sequence will perform create interactions with the provided json
+    QuestionnaireResponse resources. The server SHALL be capable of creating a
+    QuestionnaireResponse resource using the create interaction.
+
+    ## Update
+    This test sequence will perform update interactions with the provided json
+    QuestionnaireResponse resources. The server SHALL be capable of creating a
+    QuestionnaireResponse resource using the update interaction.
+          )
+    id :questionnaire_response_group
+    run_as_group
+
+    test from: :questionnaire_response_patient_search
+    test from: :questionnaire_response_context_search
```

Comment:
This test is listed as optional, but the capability statement requires the context parameter, so it should be required

Reply:
Addressed in [Correct IG links, wording, remove optional for some tests](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/20bdca6843a3ba2da4dfd0eb1800e5a326e5b768)